setup.py

```diff
@@ -39,6 +39,7 @@ def get_requirements():
 setup(
     name='picatrix',
     version=version.get_version(),
+    author = 'Kristinn',
```

Comment:
we already have the maintainer field set, adn that is set to:
```
 46     maintainer='Picatrix development team',
```

And although I may have committed the first code intot he project I will not be the only author, it is a team of people that will be behind this project, so a more correct author statement would be the `Picatrix Development Team`